docker-compose.yml

```diff
@@ -25,7 +25,9 @@ services:
     <<: *base_django_setup
     ports:
       - 8000:8000
-    command: python manage.py runserver 0.0.0.0:8000
+    command: python manage.py runserver 0.0.0.0:8000 --noreload
```

Comment:
What does the `--noreload` flag mean?

Reply:
Stop autoreload if you make changes in dev